packages/aws-cdk-lib/aws-lambda/README.md

```diff
@@ -967,7 +967,7 @@ managing concurrency.
 
 ## Lambda with SnapStart
 
-SnapStart is currently supported only on Java 11/Java 17 runtime. SnapStart does not support provisioned concurrency, the arm64 architecture, Amazon Elastic File System (Amazon EFS), or ephemeral storage greater than 512 MB. After you enable Lambda SnapStart for a particular Lambda function, publishing a new version of the function will trigger an optimization process.
+SnapStart is currently supported only on Java 11/Java 17 runtime. SnapStart does not support provisioned concurrency, Amazon Elastic File System (Amazon EFS), or ephemeral storage greater than 512 MB. After you enable Lambda SnapStart for a particular Lambda function, publishing a new version of the function will trigger an optimization process.
```

Comment:
SnapStart is also available on Java 21. 

You might want to flip this around to say "SnapStart is not available on Java 8 (AL1 or AL2)" or words to that effect.

Reply:
We need to mention that it's only supported **in Java**, so we can't leave "just Java8" out of the support.  (Yes, @msailes, there are other runtimes other than Java 😉)

The [official documentation](https://docs.aws.amazon.com/lambda/latest/dg/snapstart.html) mentions "SnapStart supports Java 11 and later Java managed runtimes", so we can go with something like that.